docker-compose.yml

```diff
@@ -1,10 +1,162 @@
+# This docker-compose file will stand up the entire stack including ingress and inventory.
 version: "3"
 services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_SERVER_ID=1
+      - KAFKA_OPTS='-Dzookeeper.admin.enableServer=false'
+  kafka:
+    image: confluentinc/cp-kafka
+    ports:
+      - 29092:29092
+    depends_on:
+      - zookeeper
+    environment:
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:29092
+      - KAFKA_BROKER_ID=1
+      - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:32181
+  puptoo:
+    image: quay.io/cloudservices/insights-puptoo:latest
+    command:
+      - python
+      - -m
+      - puptoo.app
+    ports:
+      - 8000:8000 #for prometheus endpoint
+    environment:
+      - REJECTION_TOPIC=platform.upload.validation
+      - LOGLEVEL=INFO
+    depends_on:
+      - kafka
+  minio:
+    image: minio/minio
+    command: server /data
+    volumes:
+      - /data
+    ports:
+      - 9000:9000
+    environment:
+      - MINIO_ACCESS_KEY
```

Comment:
It wouldn't hurt to note somewhere you need to export this variables to the environment previous to bring the entire thing up. As this is meant for a local development environment, maybe we can just set a default value for both the access and the secret keys ? [Ingress already does this](https://github.com/RedHatInsights/insights-ingress-go/blob/master/.env#L3-L4)

Reply:
+1 It would be nice to have a `.env.example` file as well with defaults set and mentioned in the readme to copy or touch a empty `.env`.